src/gui/processing/qgsprocessingwidgetwrapperimpl.cpp

```diff
@@ -1258,23 +1258,11 @@ QgsProcessingDurationParameterDefinitionWidget::QgsProcessingDurationParameterDe
   mDefaultLineEdit = new QLineEdit();
   vlayout->addWidget( mDefaultLineEdit );
 
-  vlayout->addWidget( new QLabel( tr( "Default unit type" ) ) );
```

Comment:
Why wouldn't we keep this?

Reply:
@nyalldawson , hm, was I feverish last night?! Reverting.